static/503.html

```diff
@@ -0,0 +1,45 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <title>Code City</title>
```

Comment:
503 - Code City
(to match the existing "404 - Code City" page)

Reply:
Done.  I made some other changes for consistency as well.  Previously I had tried to make the 503 page look like the login page, but now it is much more like the 404 page.